examples/nidaqmx/analog-input-every-n-samples.py

```diff
@@ -55,49 +55,29 @@ async def _main():
         try:
             client = grpc_nidaqmx.NiDAQmxStub(channel)
 
-            async def raise_if_error(response):
-                """Raise an exception if an error was returned."""
-                if response.status:
-                    response = await client.GetErrorString(
-                        nidaqmx_types.GetErrorStringRequest(error_code=response.status)
-                    )
-                    raise Exception(f"Error: {response.error_string}")
-
-            async def raise_if_error_async(awaitable_call):
-                """Await response, then raise an exception if an error was returned."""
-                response = await awaitable_call
-                await raise_if_error(response)
-                return response
-
-            create_response: nidaqmx_types.CreateTaskResponse = await raise_if_error_async(
```

Comment:
Before: this assigned a local variable with a type annotation.
```
create_response: nidaqmx_types.CreateTaskResponse = ...
```

After: this incorrectly overwrites the type?
```
nidaqmx_types.CreateTaskResponse = ...
```

Reply:
Whoops, didn't mean to remove the `create_response: `. I've fixed this.